pkg/sql/logictest/testdata/logic_test/alter_table

```diff
@@ -820,6 +820,11 @@ ROLLBACK
 statement ok
 CREATE TABLE audit(x INT); ALTER TABLE audit EXPERIMENTAL_AUDIT SET READ WRITE;
 
+query T
+SELECT feature_name FROM crdb_internal.feature_usage WHERE feature_name = 'sql.schema.set_audit_mode.READ WRITE'
```

Comment:
I think its idiomatic for these to not have spaces

Reply:
hmm, i've seen some with, e.g. the comparator has `sql.cmp.int < int`

Reply:
oh hmm. Idk, looks nicer in this case at least :D